tools/topology/topology2/cavs-mixin-mixout-ctc-ssp.conf

```diff
@@ -0,0 +1,100 @@
+#
+# Pipeline definitions
+#
+# PCM0 ---> gain ----> Mixin ----> Mixout ----> gain ----> CTC ----> SSP0
+# PCM1 ---> gain ----> Mixin ----> Mixout ----> gain ----> CTC ----> SSP1
```

Comment:
Do we need 2x gains ? I dont know the usage so I could be mistaken. Maybe best to explain usage in a comment above.

Reply:
Thanks!. Add a comment.